src/idc/idc.c

```diff
@@ -349,8 +349,15 @@ static int idc_ppl_state(uint32_t ppl_id, uint32_t phase)
 
 	if (phase & IDC_PPL_STATE_PHASE_TRIGGER) {
 		bool delayed = false;
+		int ret;
+
+		ret = ipc4_pipeline_trigger(ppl_icd, cmd, &delayed);
+		if (delayed)
+			tr_err(&idc_tr,
+			       "[NOT_RIGHT] idc: comp %d: cmd %d is delayed!!",
```

Comment:
I would change to "error"

Reply:
It has not trigger ed in the first test plan run, I will trigger a new one to see if it ever happens.
In theory the likeliness of this check to trigger should be the same as we have delayed trigger in the primary core.